Neos.ContentRepository.Core/Classes/ContentRepository.php

```diff
@@ -247,6 +244,19 @@ public function getContentGraph(WorkspaceName $workspaceName): ContentGraphInter
         return $this->projectionState(ContentGraphFinder::class)->getByWorkspaceName($workspaceName);
     }
 
+    public function findWorkspaceByName(WorkspaceName $workspaceName): ?Workspace
```

Comment:
Some doc comment would be nice, e.g:
```suggestion
    /**
     * Returns the workspace with the given name, or NULL if it does not exist in this content repository
     */
    public function findWorkspaceByName(WorkspaceName $workspaceName): ?Workspace
```